endpoint/packages/user/__init__.py

```diff
@@ -1,23 +1,20 @@
-from flask_login import LoginManager
-from model import User, Group, Token
 from views import bp
 from rest import bp as rest_bp
-from endpoint.database import db
-from admin_views import AuthModelView, AdminUserView, AdminGroupView, AdminTokenView
+from flask_login import LoginManager
+from endpoint.data_backends import DataLayer
 
 
 login_manager = LoginManager()
 
 
-def init(app, admin):
+def init(app):
+    # DataLayer.load_api(app.config["DATASTORE"])
+    # DataLayer.api.init(app)
```

Comment:
Are these commented out for debugging or should they deleted or uncommented?

Reply:
That is dead code, will remove. Thanks!